### DISCUSSION TITLE
how to setting auto delete old data?

### DISCUSSION THREAD

**Discussion by haidarabdillah** [blockscout/blockscout#10287] [Q&A]
i want to use blockscout for indexing transactions only, then save the transaction details of my whitelist address to other database, due to limitation of disk, i want to set auto delete old data, maybe more than 1 week ago, any feature similiar to setting this?

**Comment by phpmac** [comment]
blocks

**Comment by vbaranov** [comment]
> i want to use blockscout for indexing transactions only, then save the transaction details of my whitelist address to other database, due to limitation of disk, i want to set auto delete old data, maybe more than 1 week ago, any feature similiar to setting this?

There is no such feature at the moment. Though, this is possible to do it in a semi-manual manner:
- to set a range of blocks to index for catchup indexer via `FIRST_BLOCK`, `LAST_BLOCK` or `BLOCK_RANGES` env variables. See more info on indexer variables in [the docs](https://docs.blockscout.com/for-developers/information-and-settings/env-variables/backend-env-variables#indexer-management). Also, it worth disabling real-time indexer at the same time via `DISABLE_REALTIME_INDEXER`)
- index that range
- process the data
- clear the Blockscout DB
- and repeat with another range of blocks.
